src/System.Memory/ref/System.Memory.cs

```diff
@@ -226,7 +226,7 @@ public abstract partial class OwnedMemory<T> : System.Buffers.IRetainable, Syste
         public abstract System.Span<T> Span { get; }
         public void Dispose() { }
         protected abstract void Dispose(bool disposing);
-        public abstract System.Buffers.MemoryHandle Pin(int offset=0);
+        public abstract System.Buffers.MemoryHandle Pin(int byteOffset=0);
```

Comment:
Nit: Extra space around `=` ?

Reply:
There aren't spaces around `=` in most other places within the refs:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L1521
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L462
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L1480

But there is some inconsistency:
https://github.com/dotnet/corefx/blob/master/src/System.Buffers/ref/System.Buffers.cs#L16
cc @weshaggard

Reply:
Yeah the tool doesn't put spaces around this. Easy to fix if someone want to change it, https://github.com/dotnet/buildtools/blob/master/src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.Methods.cs#L159